tests/test_ddebug_canc/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+make dd
+
+if grep "archimedes.c:16" dd.line/rddmin-cmp/dd.line.exclude ; then
+  if grep "archimedes.c:17" dd.line/rddmin-cmp/dd.line.exclude ; then
+    echo "success !"
+    exit 0
+  else
+    echo "missing line 17 (cancellation)"
+    exit 1
+  fi
+else
+  echo "missing line 16 (round-off)"
```

Comment:
This test should be changed. When run with the cancellation backend, **only line 17** should be detected.